guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,7 +4,11 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== postgresql-evr extension no longer required
+
+Installation of the Katello database on remote systems where root access is not available is now possible.
+Only a basic PostgreSQL installation is required, which should enable installs on systems like Amazon RDS or Azure Database for PostgreSQL.
```

Comment:
```suggestion
Only a basic PostgreSQL installation is required.
With this feature, you can now install on systems like Amazon RDS or Azure Database for PostgreSQL.
```
I'm just trying to come up with a way to avoid using "should", which can evoke uncertainty and that's not really what we want :)

Reply:
Well, there is uncertainty since we haven't tested these external databases. If it's better to leave the mention out until it's tested, I can remove the bits mentioning "Amazon RDS or Azure Database".

Reply:
I'll reword it by keeping out the uncertainty, but also not saying yet that it's a fully-supported feature.

Reply:
Eh, but we already use "like", so we're not fully committing to saying one system is tested. I'll just use the requested changes :)